plugins/lenovo-thinklmi/fu-self-test.c

```diff
@@ -39,7 +39,9 @@ fu_test_self_init(FuTest *self, GError **error_global)
 	g_autoptr(FuProgress) progress = fu_progress_new(G_STRLOC);
 	g_autoptr(GError) error = NULL;
 
+#if GLIB_CHECK_VERSION(2, 64, 0)
```

Comment:
did the level of this message change?

Reply:
Nope, but I couldn't see how the test would pass with old versions...

Reply:
Yeah I guess makes sense.